bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/dialogs/ResourceInfoPage.java

```diff
@@ -1049,11 +1068,11 @@ public void performChange(IResource resource) {
 		};
 	}
 
-	private List/*<IResource>*/ getResourcesToVisit(IResource resource) throws CoreException {
+	private List/* <IResource> */ getResourcesToVisit(IResource resource) throws CoreException {
```

Comment:
> Would be nice if you could extract the generic comments into another commit and make it real generics instead of only fixing whitespace.

The white spaces were added unintentionally by the eclipse preferences. The commented generics were added as a part of this commit([Bug 349993 - [IDE] Allow to apply file attribute changes recursively · eclipse-platform/eclipse.platform.ui@7e4fb75 · GitHub](https://github.com/eclipse-platform/eclipse.platform.ui/commit/7e4fb75a060e47c7f39a8dfdf0afc002b9fc0835?diff=split&w=0)) on Aug 31 2011.